docs/application/dotnet/guides/nui/xaml/introduction-with-example.md

```diff
@@ -0,0 +1,147 @@
+# Example
+You can use XAML to define the layout of your user interface. To define the layout, follow these steps:
+1. Create a XAML file
+2. Load the XAML file
+3. Put the XAML file in the appropriate file directory
+
+> **Note**
+> 
+> WPF XAML provides a template to create an application easily, but NUI XAML currently doesn't have a XAML application template. However it will be opened and released at **Tizen.NET API7** version soon.
+
+## Create XAML
+You can use XAML to easily implement the layout shown in the following figure:
+
+![AmbientPage](./Pictures/AmbientPage.PNG)
```

Comment:
Just a suggestion, please ignore if invalid:

Could we change 2018 to 2019 in the image?

Reply:
Hello, 
it is just an image and just an example.
**I don't think it needs be changed.**
thank you.